tests/api/test_controller.py

```diff
@@ -2401,6 +2403,59 @@ def mock_fulfill(*args, **kwargs):
             assert "here's your book" == response.get_data(as_text=True)
             assert [] == self._db.query(Loan).all()
 
+    def test_no_drm_overdrive_fulfill(self):
+        """A very specific flow, wherein an overdrive work does not have DRM
+        for it's fulfillment. In which case we must simply redirect the client
+        to the non-DRM'd location instead doing a proxy download"""
```

Comment:
Here again, OD open books are the first example, but would prefer the naming/comments cover the more general case.